src/components/Article.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState} from "react"
+import axios from "axios"
+import { useParams } from "react-router-dom"
+
+function Article (){
+    const article = useParams()
+    const articleChoice = article.article_id
+    
+    const [oneArticle, setOneArticle] = useState([])
+     useEffect(()=>{
```

Comment:
A loading state would be good here for the user to have some indication that the page is currently processing the request